apps/functions/deadline-submissions/deadline-submission-command/index.js

```diff
@@ -0,0 +1,46 @@
+import { BlobStorageClient } from '@pins/blob-storage-client';
+import config from './config';
+import api from './back-office-api-client';
+
+const { storageUrl, submissionsContainer, uploadsContainer } = config;
+
+/**
+ *
+ * @param {import('@azure/functions').Context} context
+ * @param {import('@pins/api/src/message-schemas/commands/new-deadline-submission').DeadlineSubmission} msg
+ */
+export default async function (context, msg) {
+	context.log('Handle new deadline submission', msg);
+
+	const client = BlobStorageClient.fromUrl(storageUrl);
+
+	const destinationBlobName = `${msg.deadline}/${msg.submissionType}/${msg.blobGuid}/${msg.documentName}`;
+
+	const result = await client.copyFile({
```

Comment:
Before doing this, I think we should probably validate the message - so make sure that the case exists, the timetable item exists, the line item exists (and they're all associated).

We should also check that the file itself exists on the source container - we might be able to do this by looking at the exceptions that client.copyFile could throw.

Whatever the result, we'll need to broadcast some sort of error message back to the front office.

Reply:
I've added checks in to make sure the case, timetable item and line item all exist. I'll set up handling the case where `copyFile` throws, and publishing error messages to service bus in a separate PR.